scripts/change-log-generator.js

```diff
@@ -63,15 +63,19 @@ const PACKAGES = 'PACKAGES';
 const RELEASE_REGEX = new RegExp(/^origin\/release\/v\d{2}\.\d{1,2}\.\d/);
 const PR_REGEX = new RegExp(/(\(#\d+\))/);
 const COMMIT_REGEX = new RegExp(/^([\da-zA-Z]+)/);
+const TYPE_REGEX = new RegExp(/([a-zA-Z]+)(?:\([a-zA-Z]+\))?:/);
```

Comment:
This accounts for:
1) Type being present without scope
2) Type being present with scope
3) Type not being present at all

Reply:
Is 3 for handling commits unverified by commitizen?

Reply:
Yes - it's also to handle the situation where the merge of the PR into the branch doesn't actually use the commit as the message.